articles/openshift/oshift-trials-gs.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Getting Started Guide for UKCloud for OpenShift trials| UKCloud Ltd
+description: Provides information to get up and running with UKCloud for OpenShift trials
+services: openshift
+author: Mudasar Hussain
+
+toc_rootlink: Getting Started
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: UKCloud for OpenShift
+toc_fullpath: Getting Started/oshift-trials-gs.md
+toc_mdlink: oshift-trials-gs.md
+---
+
+# Getting Started Guide for UKCloud for OpenShift trials
+
+## Overview
+
+Welcome to your Red Hat&trade; OpenShift on UKCloud trial. Your trial OpenShift cluster is a secure, single-tenant private deployment on UKClouds assured OpenStack cloud which avoids any noisy neighbour issues. During an OpenShift trial the OpenShift cluster is typically only connected to the internet, however connectivity to many other Government Community Networks is available in OpenShift clusters post trial phase. The purpose of an OpenShift trial is for customers to get hands on with Red Hat&trade; OpenShift container orchestration technology to see if this is the correct technology for them.
+
+### Intended audience
+
+This guide is intended for developers who have signed up for a OpenShift trial with UKCloud. Developers are expected to have a good understanding of containerisation and container orchestration technologies to maximise their benefit from an OpenShift trial.
+
+## Introduction to UKCloud for OpenShift trials
+
+OpenShift is a platform as a service, this means that you no longer need to worry about the underlying hardware or operating system, instead you can focus on developing, deploying, scaling and monitoring your applications in OpenShift which maximises your returns on investment both monetary and time.
+
+With UKCloud OpenShift you have admin access to OpenShift giving you the control you need to create, deploy and administer your cluster as you see fit.
+
+All OpenShift deployments get a starter pack which includes the management and orchestration planes of OpenShift. These are essentially 3 Master nodes, 2 Infrastructure nodes and load balancers for the control and data planes.
```

Comment:
Suggest:

All OpenShift deployments get a starter pack, which includes the OpenShift management and orchestration planes. These are essentially three master nodes, two infrastructure nodes and load balancers for the control and data planes.

Reply:
Still refers to starter pack instead of Foundation Pack. Is this intentional? In the service definition, we refer to them as Foundation Packs. 
https://assets.digitalmarketplace.service.gov.uk/g-cloud-10/documents/92406/248991245857940-service-definition-document-2018-11-12-1732.pdf